packages/toast/src/ToastContext/ToastReducer/useToastReducer.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { act } from 'react-test-renderer';
```

Comment:
if this is a change that needs to be made for things to work, is this going to cause issues when merging with react 18?

Reply:
Definitely needs to be made. I don't think it should cause issues with React 18 (but we'll cross that bridge later)